shell/browser/browser.h

```diff
@@ -98,6 +100,12 @@ class Browser : public WindowListObserver {
 
   base::string16 GetApplicationNameForProtocol(const GURL& url);
 
+#if !defined(OS_LINUX)
+  // get the name, icon and path for an application
+  v8::Local<v8::Promise> GetApplicationInfoForProtocol(const GURL& url,
```

Comment:
By convention, `v8::Isolate* isolate` (&`gin::Arguments* args`) should go first in a parameter list

Reply:
thank you!